tests/unit/test__helpers.py

```diff
@@ -452,6 +452,99 @@ def test_w_bogus_string_value(self):
             self._call_fut("12:12:27.123", object())
 
 
+class Test_range_from_json(unittest.TestCase):
+    def _call_fut(self, value, field):
+        from google.cloud.bigquery._helpers import _range_from_json
+
+        return _range_from_json(value, field)
+
+    def test_w_none_nullable(self):
+        self.assertIsNone(self._call_fut(None, _Field("NULLABLE")))
+
+    def test_w_none_required(self):
+        with self.assertRaises(TypeError):
+            self._call_fut(None, _Field("REQUIRED"))
+
+    def test_w_wrong_format(self):
+        range_field = _Field(
+            "NULLIBLE",
```

Comment:
s/NULLIBLE/NULLABLE/ throughout this file.

Reply:
Thanks for catching this, I just replaced all of them